web-ui/src/main/resources/catalog/components/edit/onlinesrc/partials/remote-record-selector.html

```diff
@@ -28,13 +28,18 @@
     </div>
   </form>
 
-  <div class="list-group" data-ng-if="remoteRecord.title">
+  <div class="list-group" data-ng-if="remoteRecord.uuid">
     <li
       class="list-group-item"
       data-ng-click="updateSelection();triggerSearch();"
       data-ng-class="{'active' : selectionList[0].uuid === remoteRecord.uuid}"
     >
-      {{remoteRecord.title}} ({{remoteRecord.uuid}})
+      <input
+        type="text"
+        class="form-control"
+        data-ng-model="remoteRecord.title"
+        placeholder="{{'remoteAssociatedMetadataTitlePlaceholder' | translate}}"
+      />
```

Comment:
```suggestion
      /> ({{remoteRecord.uuid}})
```
to keep a reference to the URL ?